app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
+require "application_responder"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.